app/dashboard/templates/bulk_payout_bounty.html

```diff
@@ -201,6 +201,13 @@ <h5>{% trans 'Payout Preview' %}</h5>
     {% include 'shared/footer.html' %}
     {% include 'shared/messages.html' %}
   </body>
+
+  <!-- jQuery -->
```

Comment:
this is already loaded in footer_scripts.js

Reply:
got it; ill factor it back into footer_scripts

Reply:
cool I will wait until you finish to review it, ping me when ready

Reply:
just updated 1eb3739d21bc4c23760d2eae6fdcf96cc8d5bdab